### PR TITLE
Remove materialize.min.css

### PR DESCRIPTION
**Pull request by Misiu**
I've removed the reference to `materialize.min.css` in both layouts (btw why do we need two base layouts??)
I've also added a variable for footer color.
In the next PR I'd like to cleanup CSS, extract colors into variables and try to add dark theme (based on colors from HA Frontend)
Closes https://github.com/esphome/dashboard/issues/174

### CODE REVIEW

**Comment by glmnet** [comment]
If you mean the new base layout is because of the other app ESPHome-web.

**Comment by Misiu** [comment]
I didn't check that one with my changes.
I'm on windows and it's hard to run dev env with scripts.
Ideally, this should be done via npm run, but I can work on that in the future.
@glmnet could you please check if my changes don't affect ESPHome-web?

File: raw_package/base.template.html [hunk 1]
```diff
@@ -20,18 +20,13 @@
...
     <link
       rel="stylesheet"
       href="{{ get_static_file_url('css/esphome-2.css') }}"
     />
   </head>
 
   <body>
-    {% block main %}{% end %}
-    {% block footer %}{% end %}
+    {% block main %}{% end %} {% block footer %}{% end %}
```
Comment (balloob) [inline raw_package/base.template.html:30]: Please revert this change.

**Comment by balloob** [comment]
We have 2 base layouts because ESPHome and ESPHome Web use different template engines which prevents us from using multiple inheritance. Ideally we would migrate both to the same.

File: raw_package/base.template.html (hunk 1 above)
Comment (Misiu) [reply raw_package/base.template.html:30]: prettier formatted this. Give me a sec and I revert it.

Comment (Misiu) [reply raw_package/base.template.html]: done
